app/views/admin/shelters/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render partial: 'admin/header', locals: { name: "Shelters" } %>
+<h1>All Shelters</h1>
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
+    <h3><%= shelter.name %></h3>
+    <p>Created at: <%= shelter.created_at %></p>
+    <%= link_to "Update #{shelter.name}", "/shelters/#{shelter.id}/edit" %>
+    <%= link_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
```

Comment:
Just a heads up that this `delete` doesn't work on the deployed site because you need to disable Turbo! Or could swap this for a button if you don't want to deal with Turbo nonsense